### DISCUSSION TITLE
Make `maxConcurrentUploads` configurable

### DISCUSSION THREAD

**Discussion by Emily3403** [ente-io/ente#4271] [Enhancements]
It would be great if the hardcoded value [`maxConcurrentUploads`](https://github.com/ente-io/ente/blob/ca3d795a807aed6ad2a16692b44e76c4fb2478b1/web/apps/photos/src/services/upload/uploadManager.ts#L88) could be somehow adjusted, either by providing a command line switch, in the `~/.config/ente` directory or, preferably, with a UI option to manually override this. 
The "Advanced" option already exists within the preferences and I would suggest adding a new input field for number of concurrent uploads.

With this option, the ["rough heuristic" of 4 concurrent uploads](https://github.com/ente-io/ente/discussions/2145#discussioncomment-9779769) could be improved by the user, depending on their system and requirements. 

For me in particular, this is a big bottleneck and my setup could easily handle 8+ concurrent uploads.

**Comment by AlejandroAkbal** [comment]
Same here, wish it was made configurable
